ambry-router/src/main/java/com.github.ambry.router/PutManager.java

```diff
@@ -245,9 +247,14 @@ void onComplete(PutOperation op) {
     if (e != null) {
       blobId = null;
       routerMetrics.onPutBlobError(e);
-      routerCallback.scheduleDeletes(op.getSuccessfullyPutChunkIdsIfComposite());
+      routerCallback.scheduleDeletes(op.getSuccessfullyPutChunkIdsIfComposite(), op.getServiceId());
     } else {
-      notificationSystem.onBlobCreated(op.getBlobIdString(), op.getBlobProperties(), op.getUserMetadata());
+      CreatedBlobType blobType = op.getNumDataChunks() == 1 ? CreatedBlobType.Simple : CreatedBlobType.Composite;
```

Comment:
This is how it is done in elsewhere in PutManager/PutOperation. See `updateChunkingAndSizeMetricsOnSuccessfulPut()`